src/lucky.cr

```diff
@@ -22,6 +22,7 @@ require "./lucky/paginator/*"
 module Lucky
   Habitat.create do
     setting logger : Dexter::Logger
+    setting action_pipe_log_level : ::Logger::Severity? = nil, example: "Logger::Severity::INFO"
```

Comment:
What does `example` mean here? I think I could guess but I've never seen it. 😄

Reply:
`example` is what gets displayed in the error message when it's not set. But since this is nilable, does it throw an error?

Reply:
Yeah it shows up if you forget to set it, but Jeremy is right that you'll never see it :P

I think I will leave it though since in the future I want to add a task to Habitat that will show the options available, examples, and also where the configuration is being set. It'll be pretty sick I think